tests/terraform/graph/checks/resources/APIProtectedByWAF/main.tf

```diff
@@ -1,4 +1,14 @@
-resource "aws_api_gateway_rest_api" "pass" {
+resource "aws_api_gateway_rest_api" "regional" {
+  name = var.name
+
+  policy = ""
+
+  endpoint_configuration {
+    types = ["REGIONAL"]
+  }
+}
+
+resource "aws_api_gateway_rest_api" "edge" {
```

Comment:
I think you forgot to change the endpoint type here 🙂

Reply:
True! That's embarrassing) Fixed